browser-test/src/support/admin_programs.ts

```diff
@@ -160,11 +160,17 @@ export class AdminPrograms {
 
   // TODO(clouser): More fine-grained selectors for this and active.
   async expectDraftProgram(programName: string) {
-    await this.page.isVisible(this.programCardSelector(programName, 'Draft'))
+    expect(
```

Comment:
Thanks for fixing! That was a complete miss when I added these. I assume that I thought `isVisible()` would wait UNTIL something was visible, not return the current visibility state :(.